python/pyspark/sql/connect/streaming/query.py

```diff
@@ -24,6 +25,9 @@
 from pyspark.sql.streaming.query import (
     StreamingQuery as PySparkStreamingQuery,
 )
+from pyspark.errors.exceptions.connect import (
+    StreamingQueryException as CapturedStreamingQueryException,
```

Comment:
why need this?

Reply:
To align with what is done in sql/streaming/query: https://github.com/apache/spark/blame/master/python/pyspark/sql/streaming/query.py#L25